docs/theming.md

```diff
@@ -292,6 +292,8 @@ mwc-button {
 }
 ```
 
+Note: the above selector will not select mwc-buttons enclosed in the shadow roots of other custom elements. [CSS Custom Properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) can cascade through shadow root boundaries, but selectors cannot. In order to style mwc-buttons inside of other components' shadow roots, you must select the component that encloses the mwc-button or one of the component's ancestors which will apply the above custom properties to that sub/tree.
```

Comment:
lol I said "sub/tree" so you would chose one. Please change to probably "subtree"

Reply:
Done.